src/NorthwindWebsite/Presentation/Controllers/RolesController.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using NorthwindWebsite.Business.Models;
+using NorthwindWebsite.Infrastructure.Entities;
+
+namespace NorthwindWebsite.Presentation.Controllers;
+
+[Authorize(Roles = "Admin")]
+[Route("Admin/Roles/")]
```

Comment:
Is `/` at last position of route is necessary?
And why do you need this `Admin` prefix at the beginning? You can remove, because, if there is going to be a new role that could handle role operation, it would be inconsistent.